cmd/autotestgridgenerator/main.go

```diff
@@ -98,9 +99,26 @@ func main() {
 		logrus.Info("no new commits, exiting ...")
 		os.Exit(0)
 	}
+	newRepo := githubRepo
```

Comment:
This is not a `newRepo`. I can understand that this value can change in the below code and become a new repo but now it is not.

Reply:
I'll change the name to forkedRepoName. Please correct me if thats not what you meant